### PR TITLE
Universal/doc 3399 rename to closed plat

### PR DESCRIPTION
**Pull request by jpriyas**
This PR removes the old text that listed the NDA platforms and their package names explicitly.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/Building-For-Consoles.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-- **PlayStation 5**: com.unity.render-pipelines.ps5
-- **Xbox One**: com.unity.render-pipelines.xboxone
-- **Game Core Xbox Series**: com.unity.render-pipelines.gamecore
-- **Game Core Xbox One**: com.unity.render-pipelines.gamecore
+If you have a license to develop games for Closed platforms that require you to meet the confidentiality and legal agreements of the platform provider, then see the relevant developer forums for a link to the console specific render pipeline package.
 
 ## Platform package installation
 
-To install a platform package, download it from the relevant platform developer forum (you cannot get these packages from the package registry or Package Manager). To do this:
+To install a platform package, download it from the relevant platform developer forum. Note that you cannot get these packages from the package registry or Package Manager. To do this:
```
Comment (oleks-k) [inline com.unity.render-pipelines.universal/Documentation~/Building-For-Consoles.md:7]: My suggestion:

Closed platform packages are not available in the package registry and the Package Manager.

To install a closed platform package:

1. Download the package from the relevant platform developer forum.

2. Use the Package Manager to install the package locally. For information on how to install packages locally, see [Installing a local package](https://docs.unity3d.com/Manual/upm-ui-local.html).

Comment (jpriyas) [reply com.unity.render-pipelines.universal/Documentation~/Building-For-Consoles.md:7]: Good point -- I'll implement your suggestions.

**Review by erikabar — APPROVED** [review]
Reviewed doc changes, looks good.

**Review by oleks-k — APPROVED** [review]
Looks good!

**Pull request merged**